docs/hassbian_config.md

```diff
@@ -4,16 +4,22 @@ This command is a package handler for the Hassbian scripts. All interactions for
 #### Usage
 The hassbian-config command is invoked with:
 ```bash
-hassbian-config *command* *suite*
+hassbian-config *command* *suite* *flag(optional)*
 ```
 where command is one of:
-- `install`
-- `upgrade`
-- `show`
-- `log`
-- `share-log`
+- `install` Use this to isntall an suite.
+- `upgrade` Use this to upgrade an installed suite.
+- `show` This will show you all available suites.
+- `log` This wil show you the log of last hassbian-config operation.
+- `share-log` This wil generate an hastebin link of the last hassbian-config operation.
+
+optional flags:
+- `-y` This will accept defaults on scripts that allow this.
+- `-f` This will force run an script, usefull is you try to install something a second time.
```

Comment:
Recommend rephrasing this a bit.
"This will force run an script. This is useful if you need to reinstall a package."